src/pages/components/Drawer/index.tsx

```diff
@@ -18,6 +18,8 @@ import MailIcon from "@mui/icons-material/Mail";
 import MenuIcon from "@mui/icons-material/Menu";
 import StarIcon from "@mui/icons-material/Star";
 import { IconButton, Toolbar } from "@mui/material";
+import { deepOrange, lightBlue } from "@mui/material/colors";
```

Comment:
Delete this colors import, let's import only from the theme file. lightBlue[800] could be switched to theme.palette.primary.light